resources/function_help/json/$x

```diff
@@ -2,7 +2,7 @@
   "name": "$x",
   "type": "function",
   "groups": ["GeometryGroup"],
-  "description": "Returns the x coordinate of the current feature.",
+  "description": "Returns the x coordinate of the current point feature.",
```

Comment:
```suggestion
  "description": "Returns the x coordinate of the current point feature. If the feature is a multipoint feature, then the x-coordinate of the first point will be returned.",
```

Reply:
@nyalldawson One of the issues addressed in the pr is the "x coordinate" vs "x-coordinate" use in the help. Can you align your suggestion, unless that's the right spelling? Thanks.

Reply:
It's subjective, but I think hyphenation of these terms is correct. See eg https://www.chicagomanualofstyle.org/qanda/data/faq/topics/HyphensEnDashesEmDashes.html?page=1 where it talks specifically about X/y axis, but I believe the same principal applies to coordinates.